### PR TITLE
Assign members

### PR DESCRIPTION
**Pull request by benR0gers**
using a angular-multiselect dropdown dependency, will need to do npm install before running this.
adding/removing members to/from boards is working. 
there is now a new service and an updated component for assigning members, as well as modifications to the app.module for use of the multi-select dropdown thing.

### CODE REVIEW

File: octo-app/src/app/_model/SystemUser.ts [hunk 1]
```diff
@@ -4,9 +4,14 @@ export class SystemUser {
     password: string;
     firstName: string;
     lastName: string;
-    role: number;
+    role: {id: number, name: string};
     enabled: boolean;
     credentialsNonExpired: boolean;
     accountNonExpired: boolean;
     accountNonLocked: boolean;
+    boardUserJoins: {
```
Comment (jpwrunyan) [inline octo-app/src/app/_model/SystemUser.ts]: what is boardUserJoins doing here? I can't see why it's necessary and it should not be. TODO: delete this.

**Review by jpwrunyan — CHANGES_REQUESTED** [review]
boardUserJoins on SystemUser should be deleted. I can't see that it's being used in the code anyway.

**Comment by benR0gers** [comment]
latest commit = "deleted unecessary boardUserJoins field from SystemUser class"... i made that change before I realized the JsonIgnore on the BoardUserJoins, forgot to undo, should be deleted now.

**Review by drunkenoctopus2017 — APPROVED** [review]
changes look ok. the changes on app-module made me pause, though. approved.

**Pull request merged**